src/backend/api/reviews.js

```
@@ -0,0 +1,64 @@
+const express = require("express");
+const { limit } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+
+router.get('/', async (req, res) => {
+    try {
+        const allReviews = await knex('review').select('*');
+        res.json(allReviews);
+    } catch (error) {
+        throw error
```

Nitpick:
This is a bit of an anti-pattern. If you just re-throw you can remove the try/catch since it already throws.
Rather, you might want to log the error and then throw so at least you have the info about what went wrong in the server logs.

If you want to go further, it's a great idea to not expose server details to a user. You could throw a new error like this:
`throw new Error('Something went wrong internally');`

But even better would be if the server gave back a proper HTTP response, such as:
`res.status(500).send('Internal server error');`